library_generation/test/generate_library_integration_test.sh

```diff
@@ -54,16 +50,18 @@ done
 
 mkdir -p "${output_folder}"
 pushd "${output_folder}"
-# checkout the master branch of googleapis/google (proto files) and WORKSPACE
-echo "Checking out googlapis repository..."
-# sparse_clone will remove folder contents first, so we have to checkout googleapis
-# only once.
-sparse_clone https://github.com/googleapis/googleapis.git "google grafeas WORKSPACE"
-pushd googleapis
-cp -r google "${output_folder}"
-cp -r grafeas "${output_folder}"
+if [[ ! -f "WORKSPACE" ]] || [[ ! -d "google" ]] || [[ ! -d "grafeas" ]]; then
+  echo "necessary files/folders from googleapis not found in ${output_folder}"
+  echo "will now manually download googleapis"
+  download_googleapis_files_and_folders "${output_folder}"
+fi
 # parse version of gapic-generator-java, protobuf and grpc from WORKSPACE
-gapic_generator_version=$(get_version_from_WORKSPACE "_gapic_generator_java_version" WORKSPACE "=")
+
+# temporarily override version until googleapis-gen gets updated with the latest
+# generator changes
+
+# gapic_generator_version=$(get_version_from_WORKSPACE "_gapic_generator_java_version" WORKSPACE "=")
+gapic_generator_version="2.29.0"
```

Comment:
googleapis-gen is set to 2.29.0, this is a temporary workaround to solve the diffs in the IT

Context in cl/586734351 (2.30 got rolled back to 2.29)

Reply:
2.31 got merged in googleapis-gen
https://github.com/googleapis/googleapis-gen/commit/a10ed6a77676c37e60799098d48d0afb16008613